src/OpenTelemetry.Contrib.Extensions.AzureMonitor/OpenTelemetry.Contrib.Extensions.AzureMonitor.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;net5.0</TargetFrameworks>
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj#L5 need tagprefix to version correctly.

Reply:
change to netstandard20 as well.

Reply:
these comments must be addressed by the new owner *before* actual release (the original contributor is no longer working on this so @reyang and myself will own this).